src/lib/constants.js

```diff
@@ -1,4 +1,6 @@
-export const CHANNEL_NAME = 'ThemesRFun';
+export const CHANNEL_NAME = 'Themer';
```

Comment:
Spoiler alert: Is this going to be the name? #23

Reply:
95% likelihood